res/menu/comment_selection_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/quote"
+        android:icon="@drawable/icon_quote"
```

Comment:
Yes on older API where the actions menu works like toolbar with icons. (I tested it on KitKat)